### PR TITLE
fix error for  goal cost angle

### PR DESCRIPTION
**Pull request by sszwfy**

### CODE REVIEW

**Comment by AtsushiSakai** [comment]
@sszwfy Thank you for your PR!!. I have a question about your PR. Please check the review.

File: PathPlanning/DynamicWindowApproach/dynamic_window_approach.py [hunk 1]
```diff
@@ -167,7 +167,8 @@ def calc_to_goal_cost(traj, goal, config):
     dx = goal[0] - traj[-1, 0]
     dy = goal[1] - traj[-1, 1]
     error_angle = math.atan2(dy, dx)
-    cost = abs(error_angle - traj[-1, 2])
+    cost_angle = error_angle - traj[-1, 2]
+    cost = abs(math.atan2(math.sin(cost_angle), math.cos(cost_angle)))
```
Comment (AtsushiSakai) [inline PathPlanning/DynamicWindowApproach/dynamic_window_approach.py]: What is the mathematical difference between previous one and current one?

Comment (sszwfy) [reply PathPlanning/DynamicWindowApproach/dynamic_window_approach.py]: you wellcome.
for example (error_angle=-pi,   traj[-1, 2]=pi)  =====>>>>> cost = -2pi,
but is should be zero.

**Comment by AtsushiSakai** [comment]
@sszwfy Thank you so much!!

**Pull request merged**